.github/workflows/cypress-test.yml

```diff
@@ -0,0 +1,25 @@
+name: Cypress Tests
```

Comment:
instead of running as a separate test, can you add this to [ci.yml](https://github.com/dendronhq/dendron/blob/fix/move-header-new-note/.github/workflows/ci.yml#L115:L115)? 

since we don't need it to run on every os, you can use an if clause to gate it

```yml
- name: Run cypress 
  if: runner.os == 'Linux'
  uses: cypress-io/github-action@v2
  with:
      build: yarn run cypress-build
      start: yarn cypress-start 
```

Reply:
after you do the new push, i can run the ci workflow manually from your branch which allows us to test this before merging to master